alternator/executor.cc

```diff
@@ -664,7 +664,7 @@ static future<bool> is_view_built(
 
 }
 
-static future<rjson::value> fill_table_description(schema_ptr schema, table_status tbl_status, service::storage_proxy& proxy, service::client_state& client_state, tracing::trace_state_ptr trace_state, service_permit permit)
+future<rjson::value> executor::fill_table_description(schema_ptr schema, table_status tbl_status, service::client_state& client_state, tracing::trace_state_ptr trace_state, service_permit permit)
```

Comment:
You'll need to explain in the commit message why you had to do this. These functions were perfectly happy about being non-class functions and just taking a dozen parameters that the caller gets from the class. Why did you need to change this? Or want to change this?